packages/flutter_plugin_android_lifecycle/example/android/app/src/main/java/io/flutter/plugins/flutter_plugin_android_lifecycle_example/MainActivity.java

```diff
@@ -18,9 +17,10 @@ public class MainActivity extends FlutterActivity {
   private static final String TAG = "MainActivity";
 
   @Override
-  public void configureFlutterEngine(FlutterEngine flutterEngine) {
+  public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
+    super.configureFlutterEngine(flutterEngine);
     flutterEngine.getPlugins().add(new TestPlugin());
-    flutterEngine.getPlugins().add(new IntegrationTestPlugin());
```

Comment:
Why do we not need the integration test plugin? was it dead code?

Reply:
This is a speculative change, but the fact that this wasn't calling `super` seems suspicious. Normally plugins are registered by the generated registrant.

This wasn't compiling because it's unconditionally using the integration test plugin, but that plugin is a dev dependency. If this doesn't work, I'll need to make this example app have a non-dev dependency on it instead, but that feels wrong.